.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Test with Docker
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  testing:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Setup Cache
+      uses: satackey/action-docker-layer-caching@v0.0.11
+      # Ignore the failure of a step and avoid terminating the job.
+      continue-on-error: true
+
+    - name: docker-compose up
+      run: |
+        cd docker
+        docker-compose up -d
+
+    - name: Cassandra test
+      run: |
+        docker exec jepsen-control bash -c "cd /scalar-jepsen/cassandra && lein run test --workload lwt --nodes n1,n2,n3 --ssh-private-key ~/.ssh/id_rsa"
+
+    - name: ScalarDB test
+      run: |
+        docker exec jepsen-control bash -c "cd /scalar-jepsen/cassandra && lein install"
+        docker exec jepsen-control bash -c "cd /scalar-jepsen/scalardb && lein run test --workload transfer --nodes n1,n2,n3 --ssh-private-key ~/.ssh/id_rsa"
```

Comment:
Can we also add a test for `scalardl`? Or, is there any specific reason for not doing it?

Reply:
Good point! Because the test requires the binary of ScalarDL server.